docs/_config.yml

```diff
@@ -0,0 +1,59 @@
+# For technical reasons, this file is *NOT* reloaded automatically when you use
+# 'bundle exec jekyll serve'. If you change this file, please restart the server process.
+title: Bolt
+description: >-
+  A framework that makes Slack app development fast and straight-forward. 
+  With a single interface for Slack’s Web API, Events API, and interactive features, 
+  Bolt gives you the full power of the Slack platform out of the box.
+baseurl: /bolt-py
```

Comment:
`/bolt-python`?

Reply:
Good catch! I'd also suggest `/bolt-python` for consistency with the SDK name